projects/core/components/primitive-textfield/primitive-textfield.style.less

```diff
@@ -43,6 +43,8 @@
 }
 
 .t-input {
+    .text-overflow();
```

Comment:
It would cause this when you scroll horizontally with a touchpad:
![image](https://user-images.githubusercontent.com/11832552/169288628-1f588e54-e82d-4e3f-8fe9-52b2237e645e.png)